Demo/AppDelegate.swift

```diff
@@ -18,12 +19,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Override point for customization after application launch.
         
         // Initialize the calendar store
-        let calendarVC = CalendarStoreViewController(apiKey: "0443a55244bb2b6224fd48e0416f0d9c", title: "Featured")
+        let calendarVC = CalendarStoreViewController(title: "Featured")
         
         // Show the calendar store
         window?.rootViewController = calendarVC
         window?.makeKeyAndVisible()
 
+        AnalyticsTracker.shared().launch()
```

Comment:
What happens if a partner doesn't include this call into his code? Why don't you do this inside the ctor of `CalendarStoreViewController`?

Reply:
Moved https://github.com/schedjoules/ios-public-calendars-sdk/pull/25/commits/a5c4b53717467daaef473c6d66de493d650b954d